respa_admin/static_src/styles/page-home.scss

```diff
@@ -255,4 +256,14 @@ main {
       grid-template-columns: 6fr 2fr 2fr 2fr;
     }
   }
+
+  section.invoiceable-reservations {
+    @extend .main-section;
+
+    /* Invoiceable reservation list has six columns */
+    .invoiceable-reservations ul li div.panel div.panel-body,
+    .invoiceable-reservations ul li.list-group-item.sort-bar {
+      grid-template-columns: 3fr 2fr 2fr 3fr 2fr;
```

Comment:
There are 5 grid template columns, is the comment of "six columns" correct?

Reply:
There are five columns. I'll fix the comment.

Reply:
Fixed.